### PR TITLE
Small Tweaks & Comments

### PR DESCRIPTION
**Pull request by jackspirou**
This PR adds comments and some small tweaks.

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by jackspirou** [comment]
@bradrydzewski any thoughts on this? Its just a little cleanup and comments.

File: drone/types.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ type Activity struct {
 	Link      string `json:"link_url"`
 }
 
-// Repo represents a local or remote Docker daemon that is
+// Node represents a local or remote Docker daemon that is
 // repsonsible for running jobs.
```
Comment (nlf) [inline drone/types.go]: as long as you're in here fixing comments, responsible is spelled wrong here :)

**Comment by nlf** [comment]
lgtm also, added a comment with one other little typo that could be fixed

File: drone/types.go (hunk 1 above)
Comment (jackspirou) [inline drone/types.go]: good catch!

**Comment by jackspirou** [comment]
@nlf fixed it. good catch!

**Comment by bradrydzewski** [comment]
looks good, thanks

**Pull request merged**